src/toolkits/object_detection/object_detector.hpp

```diff
@@ -58,7 +58,7 @@ class EXPORT object_detector: public ml_model_base {
   virtual void resume_training(gl_sframe data, variant_type validation_data);
   virtual void iterate_training();
   virtual void synchronize_training();
-  virtual void finalize_training();
+  virtual void finalize_training(bool use_update_metrics);
```

Comment:
This method is exposed via the C-API registration below, and I think we want it to continue to work with no arguments. For now, could you update the registration and supply a default that does compute the metrics?

In addition, I think the name `use_update_metrics` could be more clear. How about something like `disable_metrics` or `compute_final_metrics`?